content/en/docs/setup/upgrade/canary/index.md

```diff
@@ -56,6 +56,14 @@ istio-sidecar-injector          1          7m56s
 istio-sidecar-injector-canary   1          3m18s
 {{< /text >}}
 
+{{< warning >}} 
+Due to [a bug](https://github.com/istio/istio/issues/28880) in the creation of `ValidatingWebhookConfiguration`s during install, initial installations of Istio __must not__ not specify a revision. As a temporary workaround, for Istio resource validation to continue working after removing the non-revisioned Istio installation, the `ValidatingWebhookConfiguration` must be manually pointed to the revision that should handle validation.
```

Comment:
"...Istio __must not__ not specify..."  So, a revision is needed?

```suggestion
Due to [a bug](https://github.com/istio/istio/issues/28880) in the creation of `ValidatingWebhookConfiguration`s during install, initial installations of Istio __must __ specify a revision. As a temporary workaround, for Istio resource validation to continue working after removing the non-revisioned Istio installation, the `ValidatingWebhookConfiguration` must be manually pointed to the revision that should handle validation.
```

Reply:
Should be "...Istio **must not** specify", since initial install w/ revision is broken

Reply:
Gotcha. That issue description makes it look like the default installation did not have revision.